### ISSUE TITLE
Imap Agent

### ISSUE THREAD

**Issue by signallhill** [cantino/huginn#1678]
Is there an option in this agent o get reply-to in the emails?? It is not showing that field in the events.

**Comment by cantino** [comment]
I don't believe current, @signallhill.  You could add `'reply_to' => mail.reply_to` to the `create_event :payload` line in the code.
